components/pkg-dockerize/bin/hab-pkg-dockerize.sh

```diff
@@ -139,8 +139,8 @@ EXPOSE 9631 $(package_exposes $1)
 ENTRYPOINT ["/init.sh"]
 CMD ["start", "$1"]
 EOT
-  docker build --force-rm --no-cache -t $version_tag .
-  docker tag $version_tag $latest_tag
+  docker build --force-rm --no-cache -t ${version_tag,,} .
```

Comment:
Could you add a comment to explain that `${version_tag,,}` lowercases the variable? This is the first time I've even seen this pattern and I'm worried I will have forgotten it again the next time I look 😄